helm-chart/splunk-kubernetes-metrics/values.yaml

```diff
@@ -74,25 +74,17 @@ splunk:
 
 
 image:
-  heapsterTag: v1.5.1
-  sidecarTag: 1.0.1
+  name: chaitanyaphalak/k8s-new-metrics:beta
```

Comment:
move to a more official repo

Reply:
need to figure out how to to push to splunk dockerhub

Reply:
ok leave for now